apps/docs/content/guides/platform/ipv4-address.mdx

```diff
@@ -4,43 +4,94 @@ description: 'Attach an IPv4 address to your database'
 subtitle: 'Attach an IPv4 address to your database'
 ---
 
-The add-on is $4 per month - we're passing on the AWS costs to you. The add-on is available for all paid plans.
+The Supabase IPv4 add-on provides a dedicated IPv4 address for your Postgres database connection. It costs $0.0055 per hour enabled, or about $4 per month if used continuously. It can be configured in the [Add-Ons Settings](https://supabase.com/dashboard/project/_/settings/addons).
```

Comment:
```suggestion
The Supabase IPv4 add-on provides a dedicated IPv4 address for your Postgres database connection. It can be configured in the [Add-Ons Settings](/dashboard/project/_/settings/addons).
```

Let's not put pricing information in the docs (difficult to keep in sync)

Reply:
The IPv4 Add-Ons page had the pricing beforehand and if we remove the value, the only reference to its price will be directly in the Dashboard. 

I'm not opposed to taking out the information, but outside of our [changelogs](https://github.com/orgs/supabase/discussions/28438), we don't mention anywhere else that IPv4 is charged for by the hour.

It's important that it is somewhere public because using direct connections is a strategy developers can use to debug connection errors. They'll be much less likely to experiment with it if they think its a flat rate of $4.

Reply:
good context, I think we should just update the text at the source so we only have one place to correct it in future

made a PR here to merge in this one https://github.com/supabase/supabase/pull/29210, just needs frontend and kevin to look at it because dashboard/billing